httpserver-buffer.lua

```diff
@@ -0,0 +1,48 @@
+-- httpserver-connection
```

Comment:
this comment seems outdated...
Why rename the file, though? Can you explain the reasoning behind this?